SPEC.md

```diff
@@ -155,6 +155,37 @@ When persisting an output artifact to a file system, if the build tool has addit
 it should persist that metadata to a subdirectory of the directory to which the output artifact is being written of the form: ```.bom/metadata/${tool}/```.  Filenaming and subdirectory structure below that point is at the discretion of the build tool.
 ## Annex A
 
+## Annex B
+
+### Build tool selection of gitoid type
+
+GITOID_BLOB_TYPE should be set in order of precedence by:
+1. A non-empty environment variable named GITOID_BLOB_TYPE.
+2. A build tool specific flag.
+The value of GITOID_BLOB_TYPE should contain the gitoid type. It may contain multiple gitoid types separated by comma (e.g., sha1,sha256). The build tool should generate the gitoids for all specified types. The build tool should implement a default gitoid type that should be used when no flag is provided.
```

Comment:
both string form "sha1,sha256" and integer form "0,2" should be fine. The integer should be the gitoid enum value specified in the gitBOM spec.

Reply:
I feel the spec need not go into that level of detail on the option values. It is up to the tool to define appropriate values so that it can match the hash type to the values in the note section. I think strings such as sha1, sha256 are more readable and the help command can provide acceptable strings.

Reply:
Removing the need to have an option to select gitoid types. A conforming gitbom tool should generate both sha1 and sh256 gitoid types.